observations/obs_converters/GSI2DART/gsi_to_dart.f90

```diff
@@ -231,15 +242,20 @@ subroutine stop2(ierror_code)
 ! adapted from GSI/src/main/stop1.f90
 
   use kinds, only: i_kind
-  use mpisetup, only : mpi_comm_world
+  use mpisetup, only : mpi_comm_world, MPI_SUCCESS
```

Comment:
the original code seems like it should be using my_communicator and not mpi_comm_world here.